apps/evm_family/evm_txn_helpers.h

```diff
@@ -30,23 +30,22 @@
  * TYPEDEFS
  *****************************************************************************/
 
-/**
- * @brief Enum used to represent the status of payload field in a transaction.
- *
- */
 typedef enum {
-  PAYLOAD_ABSENT = 0x0,                 // No payload present in the transaction
-  PAYLOAD_SIGNATURE_NOT_WHITELISTED,    // Payload function signature is not
-                                        // recognized [Blind Signing]
-  PAYLOAD_CONTRACT_NOT_WHITELISTED,     // [OBSOLETE] Payload function signature
-                                        // is whitelisted but contract is not
-                                        // (for Transfer function) [Unverified
-                                        // Contract] [OBSOLETE]
-  PAYLOAD_CONTRACT_INVALID,    // Payload function signature and contract both
-                               // are whitelisted but doesn't match [Invalid
-                               // Transaction]
-  PAYLOAD_WHITELISTED,    // Payload is completely recognized [Clear Signing]
-} PAYLOAD_STATUS;
+  /// No data present in the transaction
+  EVM_TXN_NO_DATA = 0x0,
+
+  /// Data present in the transaction is invalid
+  EVM_TXN_INVALID_DATA,
+
+  /// Function signature in data is not recognized
+  EVM_TXN_UNKNOWN_FUNC_SIG,
+
+  /// Data is for token transfer function
+  EVM_TXN_TRANSFER_FUNC,
```

Comment:
Let's consider the name EVM_TXN_TOKEN_TRANSFER_FUNC

Reply:
renamed